android/tangram/src/com/mapzen/tangram/geometry/Geometry.java

```diff
@@ -0,0 +1,35 @@
+package com.mapzen.tangram.geometry;
+
+import java.util.Map;
+
+/**
+ * Geometry is an abstract container of LngLat points and associated properties
+ */
+public abstract class Geometry {
+
+    protected double[] coordinates;
+    protected int[] rings;
+    protected String[] properties;
```

Comment:
I think the better question to ask is "should we have a mechanism to remove individual features from the map?" Right now we don't, but I think we can add such a mechanism later in a natural and backwards-compatible way.